app/middleware/auth.js

```diff
@@ -0,0 +1,57 @@
+const jwt = require('jsonwebtoken');
+
+/*
+exports.verifyToken = (req, res, next) => {
+	try {
+		const token = req.headers.authorization.split('Bearer ')[1];
+		let jwt_secret = 'jwt-secret-key';
+
+		req.decoded = jwt.verify(token, jwt_secret);
+		return next();
+	} catch (e) {
+		if (e.name === 'TokenExpiredError') {
+			return res.status(419).json({
+				success: false,
+				message: 'Token 만료'
+			});
+		}
+		return res.status(401).json({
+			success: false,
+			message: 'Token이 유효하지 않습니다'
+		});
+	}
+}
+
+*/
+
+exports.verifyToken = (req, res, next) => {
+	const token = req.headers['x-access-token'] || req.query.token
+
+	if (!token) {
+		return res.status(403).json({
+			success: false,
+			message: '로그인이 필요합니다'
```

Comment:
Codacy has a fix for the issue: [Replace `'로그인이·필요합니다'` with `"로그인이·필요합니다",`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9684192)
```suggestion
			message: "로그인이 필요합니다",
```